talk/owt/sdk/include/cpp/owt/base/videorendererinterface.h

```diff
@@ -20,15 +20,16 @@ namespace base {
 enum class VideoBufferType {
   kI420,
   kARGB,
+  kI444,
```

Comment:
Put the newly added value as the last one?

Reply:
done.